src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -253,6 +292,23 @@ export const LedgerCryptoProvider: () => Promise<CryptoProvider> = async () => {
     }
   }
 
+  const prepareTtl = (ttl?: number) => {
+    if (!ttl && !isFeatureSupportedForVersion(LedgerCryptoProviderFeature.ALLEGRA)) {
+      throw Error(Errors.LedgerOptionalTTLNotSupported)
+    }
+    return ttl && ttl.toString()
+  }
+
+  const prepareValidityIntervalStart = (validityIntervalStart?: number) => {
+    if (
+      validityIntervalStart
```

Comment:
validityIntervalStart equal to zero would not work as it's falsy, see comment above about ttl

Reply:
Fixed